cron_test.go

```diff
@@ -1,57 +1,48 @@
 package cron
 
 import (
-	"fmt"
-	"math/rand"
-	"sort"
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/google/go-cmp/cmp"
 )
 
 // One []int each for minutes, hours, ...
 // Must be in sorted order. nil == '*'
 type testSchedule [5][]int
```

Comment:
One thing about `testSchedule` that initially threw me off was that all fields are 0-indexed, while I was expecting day of month and day of month to be 1-indexed like the cron expression. WDYT about making those fields 1-indexed?

Reply:
Good idea. Done.